packages/store/addon/-private/system/model/model.js

```diff
@@ -1193,6 +1194,10 @@ if (DEBUG) {
     init() {
       this._super(...arguments);
 
+      if (DEBUG) {
+        this._getDeprecatedEventedInfo = () => `${this.name}#${this.id}`;
```

Comment:
`name` is probably the wrong thing here, `_internalModel.modelName` is likely what was intended.